webapp/src/components/jira_epic_selector/jira_epic_selector.test.tsx

```diff
@@ -72,7 +72,7 @@ describe('components/JiraEpicSelector', () => {
         args = props.fetchEpicsWithParams.mock.calls[2][0];
         expect(args).toEqual({
             epic_name_type_id: 'customfield_10011',
-            jql: 'project=KT and issuetype=10000  and "Epic Name"~"some input"  ORDER BY updated DESC',
+            jql: 'project=KT and issuetype=10000  and ("Epic Name"~"some input" or "Epic Name"~"some input*") ORDER BY updated DESC',
```

Comment:
Does `"Epic Name"~"some input*"` not cover the former case withouth the *?

Reply:
Without the asterisk, we are using Jira's fuzzy search on the whole user input. This is especially useful when an Epic's label has multiple words. However, it does not show any results until the user has typed more than a few characters. It has different behavior depending on the search term.

Using the asterisk, the search is more strict (the Epic label must begin with the user's exact input), but the user sees results after the first keystroke. Combining these two gives the best of both worlds: Quick feedback on results, and good approximation on multiple words.

Reply:
Thanks for the explanation.